pkgs/development/libraries/sol2/default.nix

```diff
@@ -0,0 +1,36 @@
+{ fetchFromGitHub
+, lib
+, stdenv
+, cmake
+, lua
+}:
+stdenv.mkDerivation rec{
```

Comment:
I like the `finalAttrs` convention used recenty that avoids the `rec` here

Reply:
TIL